mergify_cli/ci/cli.py

```diff
@@ -0,0 +1,134 @@
+import os
+import re
+import typing
+
+import click
+
+from mergify_cli import console
+from mergify_cli import utils
+from mergify_cli.ci import junit_upload as junit_upload_mod
+
+
+ci = click.Group(
+    "ci",
+    help="Mergify's CI related commands",
+)
+
+
+CIProviderT = typing.Literal["github_action", "circleci"]
+
+
+def get_ci_provider() -> CIProviderT | None:
+    if os.getenv("GITHUB_ACTIONS") == "true":
+        return "github_action"
+    if os.getenv("CIRCLECI") == "true":
+        return "circleci"
+    return None
+
+
+def get_job_name() -> str | None:
+    if get_ci_provider() == "github_action":
+        return os.getenv("GITHUB_WORKFLOW")
+    if get_ci_provider() == "circleci":
+        return os.getenv("CIRCLE_JOB")
+
+    console.log("Error: failed to get the job's name from env", style="red")
+    return None
+
+
+def get_head_sha() -> str | None:
+    if get_ci_provider() == "github_action":
+        return os.getenv("GITHUB_SHA")
+    if get_ci_provider() == "circleci":
+        return os.getenv("CIRCLE_SHA1")
```

Comment:
Same

Reply:
Indeed it is an issue. For GH workaround is easy using the provided event path but for Circle CI it is more complicated. I think the best is to query the github pull request API but I will need to add a `github_token` in the circle ci obr inputs to do that.